src/reducer.js

```diff
@@ -2,11 +2,16 @@
 import type { Action } from 'redux';
 import type { Location } from 'history';
 import { LOCATION_CHANGED } from './action-types';
+import isEqual from 'lodash.isequal';
```

Comment:
The `fbjs` replacement would be:

`import shallowEqual from 'fbjs/lib/shallowEqual';`